ui/app/adapters/secret-engine.js

```diff
@@ -26,15 +30,43 @@ export default ApplicationAdapter.extend({
 
   createRecord(store, type, snapshot) {
     const serializer = store.serializerFor(type.modelName);
-    const data = serializer.serialize(snapshot);
+    let data = serializer.serialize(snapshot);
```

Comment:
turned to let because we redefine data after abstracting the values we need for the endpoint.